tools/meson.build

```diff
@@ -67,21 +69,23 @@ bubblejail_cli_launch = custom_target(
     command : jinja2_commands,
 )
 
-bubblejail_gui_qt_launch = custom_target(
-    'bubblejail_gui_qt_launch',
-    build_by_default : true,
-    input : files( templates_dir / 'bubblejail_gui_qt_launch.py.jinja2'),
-    depend_files : files(
-        templates_dir / 'setup_python_path.py.jinja2',
-        templates_dir / 'setup_bubblejail_settings.py.jinja2',
-    ),
-    output : 'bubblejail-config',
-    capture : true,
-    install : true,
-    install_dir : get_option('bindir'),
-    install_mode : 'rwxr-xr-x',
-    command : jinja2_commands,
-)
+if get_option('gui').enabled()
```

Comment:
BTW you probably want to disable the `.destop` from installing when GUI is disabled.

Reply:
fixed